content/en/docs/pipelines/caching.md

```diff
@@ -61,12 +61,12 @@ Kubeflow cluster.
 
     ```
     export NAMESPACE=<Namespace where KFP is installed>
-    kubectl get mutatingwebhookconfiguration cache-webhook -n ${NAMESPACE}
+    kubectl get mutatingwebhookconfiguration cache-webhook-${NAMESPACE} -n ${NAMESPACE}
```

Comment:
I believe -n is unnecessary, because it's a cluster scoped resource. Can you confirm?

Reply:
Done. Find a request reference here: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#mutatingwebhookconfiguration-v1-admissionregistration-k8s-io